src/System.Management.Automation/commands/utility/FormatAndOutput/common/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -717,7 +717,7 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
         private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Automation_ErrorRecord()
         {
             yield return new FormatViewDefinition("ErrorInstance",
-                CustomControl.Create(outOfBand: true)
+                CustomControl.Create(outOfBand: true, width: Int32.MaxValue)
                     .StartEntry()
```

Comment:
I know I've tried something similar, and the formatting system decided to pad out the width with spaces - I ended up with huge log files.

Can you see if that's still a problem?